modules/contact-form/grunion-contact-form.php

```diff
@@ -252,6 +252,7 @@ function process_form_submission() {
 		add_filter( 'contact_form_subject', array( $this, 'replace_tokens_with_input' ), 10, 2 );
 
 		$id = stripslashes( $_POST['contact-form-id'] );
+		$hash = $_POST['contact-form-hash'];
```

Comment:
Should this be sanitized somehow?  Are sha1 hashes guaranteed to be hex?  How should we do it?